phoible-references.bib

```diff
@@ -2,7 +2,7 @@
 %% http://bibdesk.sourceforge.net/
 
 
-%% Created for stiv at 2014-09-13 00:58:21 +0200 
+%% Created for stiv at 2015-02-12 10:55:53 +0100 
```

Comment:
It's preferable to have these changes to the BibTeX file in a separate PR, since they're unrelated to the ISO code checking script.  Can achieve this by having the BibTeX changes and ISO code script changes on separate branches, instead of making all changes in your local `master` branch.